readme.md

```diff
@@ -0,0 +1,69 @@
+# Frontend Cookbook
+
+## Cookbooks
+
+* [React](/cookbook-react)
+* [React Native](/cookbook-react-native)
+* [Web](/cookbook-web)
+
+## Getting started
+
+Add a `.env.development` file under `/app` with the following content:
+
+```
+REACT_APP_RECIPES_BRANCH=XXXX
+REACT_APP_GITHUB_TOKEN=XXXX
+```
+
+Where `REACT_APP_RECIPES_BRANCH` is the name of a remote branch, you can try with `master`, and `REACT_APP_GITHUB_TOKEN` is a [github personal access token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line).
+
+Install dependencies:
+
+`npm install`
+
+To start the server by default (development) move to `/app` and run:
+
+`npm run start`
+
+### New Cookbook
+
+To add a new cookbok create a `cookbook-XXX` folder where `XXX` would be the TECH name. And add a `recipes` folder inside. You can crete a base project inside to help our collaborators to test future recipes.
+
+### New recipe
+
+To add a new recipe, you need to define the recipe category first. For example, `button-1` category would be `buttons`.
+
+Under you cookbook folder, if doesn't exist already, create yor category folder, and inside your category folder create your recipe folder. Remember to check for existing categories names before to avoid having several categories with similars names!
```

Comment:
```suggestion
Under you cookbook folder, if doesn't exist already, create your category folder, and inside your category folder create your recipe folder. Remember to check for existing categories names before to avoid having several categories with similars names!
```